src/test/java/uk/gov/justice/hmpps/prison/api/resource/impl/AgencyResourceIntTest.kt

```diff
@@ -189,4 +189,114 @@ class AgencyResourceIntTest : ResourceTest() {
         .jsonPath("[0].userDescription").isEqualTo("Chapel")
     }
   }
+
+  @Nested
+  @DisplayName("GET /api/agencies/type/{type}")
+  inner class GetAgencyByType {
+    @Nested
+    inner class Security {
+      @Test
+      fun `should return 401 when user does not even have token`() {
+        webTestClient.get().uri("/api/agencies/type/{type}", "INST")
```

Comment:
I never use parameter substituion with WebTestClient - just hardcode the String e.g. `"/api/agencies/type/INST". Is there any advantage to use substituion?

Reply:
No. I will change in the next PR - thanks @mikehalmamoj